prow/proxy-presubmit.sh

```diff
@@ -45,6 +45,8 @@ fi
 
 cd $ROOT
 
+export BAZEL_TEST_ARGS="--test_output=errors --define=google_grpc=disabled"
```

Comment:
My concerns is:  test is using different code than release.  This is not testing anything.

Reply:
Just realized this is for "auth",  auth is not using grpc so it is fine then.